plugins/carbon_black_defense/komand_carbon_black_defense/actions/find_event/action.py

```diff
@@ -35,12 +35,12 @@ def run(self, params={}):
         else:
             result = requests.get(url, headers=headers)
         try:
-            data = komand.helper.clean(result.json())
+            data = insightconnect_plugin_runtime.helper.clean(result.json())
```

Comment:
yup, good point. I think I needed to make this change because of a github actions chec. but in the PR for find_event in a helper function I'll include a check for if the json call will fail if you think that should be included there or here instead?

Reply:
i think i ended up doing a check in the connection file within the call_api function - IIRC we checked during group review if that would work - still ok?